packages/rocketchat-livechat/app/client/lib/commands.js

```diff
@@ -1,4 +1,4 @@
-/* globals LivechatVideoCall, Livechat */
+/* globals LivechatVideoCall, Livechat, swal */
```

Comment:
is it still needed to add swal to globals when actually removing it?

Reply:
Livechat widget still uses sweetAlert

Reply:
:+1: makes sense I just saw the adding in a bunch of files so was checking :)

Reply:
👍